### PR TITLE
Try out new docker pipe feature

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

**Comment by RDIL** [comment]
Hey @fkorotkov what am I doing wrong here?

File: .cirrus.yml [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-  build_script: mkdocs build
-  docs_build_artifacts:
-    path: site/**
+  steps:
+    - image: squidfunk/mkdocs-material:4.4.0
+      install_script: pip install --upgrade pymdown-extensions
+      script: mkdocs build
+      docs_build_preview_artifacts:
+        path: site/**
+    - name: Deploy
```
Comment (fkorotkov) [inline .cirrus.yml]: `Unknown field name on 'Deploy' YAML node!` indicates you don't need `name` here.

Comment (RDIL) [reply .cirrus.yml]: Now it is complaining about the `skip`.  I can't deploy if its not master so not sure what to do here 🤷‍

Comment (fkorotkov) [reply .cirrus.yml]: https://github.com/cirruslabs/cirrus-ci-docs/issues/339

**Comment by RDIL** [comment]
On hold for now

**Pull request closed without merging**